### DISCUSSION TITLE
What's the difference between Scheduled Reminders and Pull Panda?

### DISCUSSION THREAD

**Discussion by andrewhampton** [community/community#22963] [New to GitHub]
<p>I Scheduled Reminders a rebranding of Pull Panda’s Pull Reminders or is there some more significant difference I’m not seeing?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/andrewhampton">@andrewhampton</a> </p>
<p>Pull Panda has been acquired by GitHub a few months ago, and their products are indeed being integrated into GitHub proper as explained on <a href="https://pullpanda.com/github" rel="nofollow noopener">https://pullpanda.com/github</a></p>

**Comment by andrewhampton** [comment]
<p>Thanks for the reply <a class="mention" href="/seveas">@seveas</a>. We currently use Pull Panda’s pull reminders at work, so my main question is should we migrate to scheduled reminders? It seems like it’s just the rebranding/integration of Pull Panda’s pull reminders. If that’s the case, I’m guessing the separate Pull Panda feature will eventually go away. Do you know if that’s the case?</p>
